selfdrive/car/subaru/carstate.py

```diff
@@ -77,6 +77,8 @@ def update(self, cp, cp_cam, cp_body):
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
       ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
+      ret.stockAeb = (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 5) or \
+                     (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert_Msg"] == 6) or ret.stockFcw
```

Comment:
why set when there's a fcw? we usually want to differentiate between AEB and FCW

Reply:
I reviewed the other brands openpilot long implementations and removed stockFcw from stockAeb